run.py

```diff
@@ -27,6 +27,46 @@ def get_config(cfg):
         raise Exception("yaml file not present")
         sys.exit()
 
+def check_test_exists(directory, test_name):
+    """
+    Checks if mentioned tests exist in particular directory or not
+    """
+    directory_path = os.path.join(os.getcwd(), 'tests') #name of directory under which test exists
+    if(test_name!= "statFile"):
+        test_file = "test_" + test_name + ".py"
+        test_file_path = os.path.join(os.path.join(directory_path, directory), test_file)
+        if not os.path.exists(test_file_path):
+            raise Exception( test_name + " test file does not exists")
+
+def check_input_validity(params):
+    """
+    Check the parameter type
+    """
+    for key , value in params.items():
+        if "number" in key.lower():
```

Comment:
This is not a very strong assumption (what if I have a `numberOfBytes: 1024`?) but I have no better ideas for the moment...